components/scanners/trufflehog/internal/transformer/transformer.go

```diff
@@ -298,12 +306,20 @@ func (t *trufflehogTransformer) mapDataSource(ctx context.Context, location Truf
 	if location.SourceMetadata.Data.Filesystem.File == "" {
 		return "", errors.Errorf("unsupported trufflehog findings with empty file entry detected, finding %#v", location)
 	}
+
+	fullPath := location.SourceMetadata.Data.Filesystem.File
+	relativePath, err := filepath.Rel(t.stripFilePathPrefix, fullPath)
```

Comment:
We could use relativePath, err := strings.TrimPrefix(fullPath, t.stripFilePathPrefix) instead?

Reply:
I think it looks fine as is, better to use a function that is made for unix paths than a plain string manipulation one